language/move-prover/spec-lang/src/ast.rs

```diff
@@ -621,3 +638,74 @@ impl<'a> fmt::Display for ExpDisplay<'a> {
         Ok(())
     }
 }
+
+// =================================================================================================
+// # AST Expression Helpers
```

Comment:
Please move this into a new module ast_helpers.

Also you like to have a type `ExpBuilder` here, then wrap the module_env inside, and make all those functions have a `&self` parameter.